src/Symfony/Bundle/WebServerBundle/composer.json

```diff
@@ -26,6 +27,9 @@
             "/Tests/"
         ]
     },
+    "conflict": {
+        "symfony/dependency-injection": "<3.3"
```

Comment:
Actually, I do not know why the DI component is no required dependency in our bundles. Is there really a use case for using one of bundles without the DI extension/config?